GCD Homework/GCD Homework/Constant.swift

```diff
@@ -8,11 +8,13 @@
 import Foundation
 
 class Constant {
+    static let getUserURL = "https://api.github.com/search/users?q=abc"
```

Comment:
```suggestion
    static let getUserURL = "https://api.github.com/search/users?q=abc+in:login"
```
Cái api này nếu search bằng users?q=abc thì nó sẽ search hết các thông tin của user cái nào có chữ abc thì nó lấy, nếu sửa thành như trên thì nó sẽ search mỗi cái "login" nào có abc thui á, có thể thử nha, nhìn sẽ trựa quan hơn